Dockerfile

```diff
@@ -63,7 +63,7 @@ RUN export ZSTD_VER=1.5.2 \
       && make install \
       && cd .. \
       && rm -r zstd-${ZSTD_VER}
-RUN export ROCKSDB_VER=7.0.4 \
+RUN export ROCKSDB_VER=7.7.3 \
```

Comment:
Can you please explain why this version was chosen?  I'm not familiar with the various versions.  A quick check on releases shows 7.7.8 is the latest in the 7.7 line, and there are 7.8, 7.9, 7.10, 8.0, 8.1 lines as well.

Reply:
this is the same version as upstream master

Reply:
Makes sense, thanks.